react/global.css

```diff
@@ -185,6 +185,10 @@ body.vtex-filter-popup-open {
   .vtex-search-result__border {
     display: none;
   }
+
+  .vtex-search-result__sidebar {
+    width: 340px;
```

Comment:
Is this necessary? A percentage or an `auto` wouldn't do the business?

Even so, use `rem` instead